src/app/service/bpa.service.ts

```diff
@@ -59,56 +62,44 @@ export class BpaService {
     /* Toaster code this.bpaService.showInfo('Remember Next time')*/
     
 
-  getServiceItems()
-  {
+  getServiceItems(){
     const getToken = localStorage.getItem('accessToken');
     const httpHeaders = {
-      headers: new HttpHeaders({
-        Accept: 'application/json',
+      headers : new HttpHeaders({
+        Accept : 'application/json',
         Authorization: `Bearer ${getToken}`
       })
     };
 
-    const url2: string = 'https://10.81.59.209:9091/bpa/api/v1.0/service-catalog/service-items?_page=1&_limit=20&status=Active&order=asc';
-
-    return this.httpClient.get(url2, httpHeaders);
+    const urlActive : string ='https://10.81.59.209:9091/bpa/api/v1.0/service-catalog/service-items?_page=1&_limit=20&status=Active&order=asc'
+    return this.httpClient.get(urlActive, httpHeaders);
   }
-  getServiceCategory(){
+
+  
+getServiceCategory(){
     const getToken = localStorage.getItem('accessToken');
     const httpHeaders = {
       headers: new HttpHeaders({
         Accept: 'application/json',
         Authorization: `Bearer ${getToken}`
       })
     };
+    const url: string = 'https://10.81.59.209:9091/bpa/api/v1.0/service-catalog/service-items?_page=1&_limit=20&status=Active&order=asc';
+    return this.httpClient.get(url, httpHeaders);
+  }
 
-    const url: string = 'https://10.81.59.209:9091/bpa/api/v1.0/service-catalog/service-categories?_page=1&_limit=200000';
-
-    return this.httpClient.get(url, httpHeaders)
-  }  
-  getDeviceList()
-  {
+  getDeviceList(){
     const getToken = localStorage.getItem('accessToken');
     const httpHeaders = {
       headers: new HttpHeaders({
         Accept: 'application/json',
         Authorization : `Bearer ${getToken}`
       })
-    };
+      };
 
     const urlDevices : string ='https://10.81.59.209:9091/bpa/api/v1.0/device-manager/devices?limit=5000&page=1&nsoInstance=RTP-Core,nso5-lsa4-re'
     
     return this.httpClient.get(urlDevices, httpHeaders);
   }
-  /* Use below approach to display Toastr from any component:
```

Comment:
Recommended to keep this code for future use.

Reply:
Already present. Some error after pulling last PR.